client/src/components/Tickets/TicketsLayout.scss

```diff
@@ -12,39 +12,33 @@
   .tickets-view {
     /* number */
     tr > *:nth-child(1) {
-      width: 7%;
-    }
-    /* subject */
-    tr > *:nth-child(2) {
-      width: 70%;
+      width: 12%;
     }
+    /* subject (fills to available width) */
     /* date */
     tr > *:nth-child(3) {
-      width: 11%;
+      width: 12%;
     }
     /* status */
     tr > *:nth-child(4) {
-      width: 12%;
+      width: 14%;
     }
   }
 }
 @media only screen and (max-width: 1679px) {
   .tickets-view {
     /* number */
     tr > *:nth-child(1) {
-      width: 12.5%;
-    }
-    /* subject */
-    tr > *:nth-child(2) {
-      width: 52.5%;
+      width: 15%;
     }
+    /* subject (fills to available width) */
     /* date */
     tr > *:nth-child(3) {
-      width: 20%;
+      width: 6.5em;
     }
     /* status */
     tr > *:nth-child(4) {
-      width: 15%;
+      width: 7em;
```

Comment:
Using `em` is fine. For an explanation of my reversal (to perfect percentages), see Footnote 2 from https://github.com/TACC/Core-Portal/pull/635#pullrequestreview-968412052.